### PR TITLE
Stable deployment Phase I

### PR DESCRIPTION
**Pull request by EndymionJkb**

### CODE REVIEW

File: pkg/deployments/tasks/20220609-stable-phase-I/readme.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# 2022-06-06 - Stable Pool Phase I
+
+Deployment of the `StablePoolFactory` to meet the Phase I milestone objectives, including a wider invariant convergence range, infrastructure improvements, and Recovery Mode, for guaranteed safe exit under emergency conditions.
+
+## Useful Files
+
+- [Polygon addresses](./output/polygon.json)
+- [Polygon mainnet address](./output/polygon.json)
```
Comment (nventuro) [inline pkg/deployments/tasks/20220609-stable-phase-I/readme.md:8]:
```suggestion
- [Ethereum addresses](./output/polygon.json)
- [Polygon addresses](./output/polygon.json)
```

File: pkg/deployments/tasks/20220609-stable-phase-I/readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+- [Arbitrum addresses](./output/arbitrum.json)
+- [Optimism addresses](./output/optimism.json)
+- [`StablePool` ABI](./abi/StablePool.json)
```
Comment (nventuro) [inline pkg/deployments/tasks/20220609-stable-phase-I/readme.md:11]:
```suggestion
- [`StablePool` ABI](./abi/StablePool.json)
- [`StablePoolFactory` ABI](./abi/StablePoolFactory.json)
```

**Comment by nventuro** [comment]
I merged master into this branch with all of the latest changes (incl #1373 and #1372). Someone should verify the build info (note that there's more than one due to multiple compiler settings).

I also deployed to all networks except Arbitrum, which is erroring with out of gas for some unknown reason. We need to do more research there.

**Comment by nventuro** [comment]
I managed to do the Arb deployment. I'm still not sure what happened - it seems like there's maybe an issue with the RPC endpoint which makes it fail to estimate gas or something. After many tries one eventually went thorugh :shrug:

**Comment by EndymionJkb** [comment]
Verified build-info at commit 2ea054378f2295dbf8eb4537ce3d90572ab1eab4

**Pull request merged**